onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -110,6 +123,12 @@ def __init__(
             else extra_options["QDQOpTypePerChannelSupportToAxis"]
         )
 
+        # Register the chosen dynamic subgraph compute quantization parameter function based on symmetric and qtype
```

Comment:
Is this part only needed in (not static)?

Reply:
Yes, only dynamic quantization (QDQ) need it.